aten/src/ATen/native/native_functions.yaml

```diff
@@ -328,6 +328,10 @@
   dispatch:
     DefaultBackend: _conj
 
+- func: Assert(bool condition, str message) -> ()
+
+- func: Assert.Tensor(Tensor condition, str message) -> ()
```

Comment:
Note: this would be the very first function in `native_functions.yaml` to start with a capital letter.

Reply:
You know what I'm going paint this bikeshed. I don't want this function to be named Assert with a capital A. Python naming convention is that all functions start with lower case and classes are upper case, and this violates that. If I had to give a suggestion, maybe try `_assert` (not `assert_` as that would imply inplace, also bad.)

Reply:
sure, moved that to https://github.com/pytorch/pytorch/pull/47763